install/oddjob/com.redhat.idm.trust-fetch-domains.in

```diff
@@ -37,7 +38,17 @@ def parse_options():
     options, args = parser.parse_args()
     safe_options = parser.get_safe_opts(options)
 
-    return safe_options, options, args
+    # We only use first argument of the passed args but as D-BUS interface
+    # in oddjobd cannot expose optional, we fill in empty slots from IPA side
+    # and filter them here.
+    trusted_domain = ipautil.fsdecode(args[0]).lower()
```

Comment:
You now longer check that args contains exactly one entry.

Reply:
I don't need to! Because we only care for the first argument after parsing the command line arguments and if `len(DNSName().labels)` is less than 2, then we exit. This covers both missing argument and wrong argument. Whether there are more than one, it is less important here than the one that we use.
Technically, the helper is always called over D-BUS by IPA framework so we control all the inputs.